src/Renderer/Simulator/NumberHelpers.fs

```diff
@@ -69,31 +69,31 @@ let textToFloatValue (text:string) =
 
     let checkNoChars (s:string) =
         s |> Seq.forall isDigitOrDot
-    
+
     match String.length text with
-    |0 -> Some 0.
-    |length ->
-        match text |> Seq.last with
-        | ch when  System.Char.IsNumber ch -> if checkNoChars text then (float text) |> Some else None
-        | last ->
+    | 0 -> Some 0.0
+    | length ->
+        match System.Double.TryParse(text, System.Globalization.NumberStyles.Float, System.Globalization.CultureInfo.InvariantCulture) with

```

Comment:
open System an System,Globalisation to make library functions without long prefixes